atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1890,8 +1890,23 @@ private void alterGcAndTombstoneInternal(
 
     public void cleanUpSchemaMutationLockTablesState() throws Exception {
         Set<TableReference> tables = lockTables.getAllLockTables();
```

Comment:
Ok, so to clarify, we should drop all except one tables in cases where multiple tables exist and then write the cleared value for that one table, correct? How do we decide on which table to pick in cases with multiple tables?